packages/tools/components-package/nps.js

```diff
@@ -62,12 +64,14 @@ const getScripts = (options) => {
 				es5: 'rollup --config config/rollup.config.js -w --environment ES5_BUILD,DEV,DEPLOY_PUBLIC_PATH:/resources/'
 			},
 			styles: {
-				default: 'concurrently "nps watch.styles.themes" "nps watch.styles.components"',
+				default: 'concurrently "nps watch.styles.themes" "nps watch.styles.components" "nps watch.styles.monitor"',
```

Comment:
watch and monitor are the same thing, something like `restart-on-add` is more descriptive

Reply:
There are a couple of general problems:
 - I still can't have it work on windows: the `ps-list` module ships a binary that simulates ps on windows, but it does not provide the full command, only the name of the executable "something.exe" without any parameters. Running `ps` on the git bash is the same.
 - Sometimes the `ps-list` fails with a promise rejection on Mac, therefore the retry. The retry works, but once I had an error trying to kill the current postcss process.